src/pages/sighting/SightingCore.jsx

```diff
@@ -143,10 +157,20 @@ export default function SightingCore({
             ? deleteAssetGroupSightingError
             : deleteSightingError
         }
+        errorTitleId={
+          vulnerableIndividual
+            ? 'REQUEST_REQUIRES_ADDITIONAL_CONFIRMATION'
+            : 'SERVER_ERROR'
+        }
```

Comment:
You can get the default `errorTitleId` when there is no vulnerable individual and still have the custom `errorTitleId` when there is by changing this to
```js
errorTitleId={
  vulnerableIndividual
    ? 'REQUEST_REQUIRES_ADDITIONAL_CONFIRMATION'
    : undefined
}
```

Reply:
I'm surprised by this. Thank you. Done.